app/matplotlib.ipynb

```diff
@@ -47,21 +43,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
    "metadata": {
-    "collapsed": true,
-    "ExecuteTime": {
-     "end_time": "2023-07-07T12:36:31.669207Z",
-     "start_time": "2023-07-07T12:36:31.661449Z"
-    }
+    "collapsed": true
    },
-   "outputs": [],
    "source": [
     "years = [2015, 2016, 2017, 2018, 2019]\n",
     "population = [100, 120, 150, 180, 200]\n",
     "\n",
-    "# write your code here"
-   ]
+    "plt.plot(years, population, label=\"Population\", color=\"blue\")\n",
+    "plt.title(\"Amount of people each years\")\n",
+    "plt.xlabel(\"Years\")\n",
+    "plt.ylabel(\"Peoples\")\n",
```

Comment:
The labels for the x and y axes and the title are generic. Consider using more descriptive labels and titles that reflect the data being plotted, such as 'Year' for the x-axis and 'Population' for the y-axis.